specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -288,6 +255,65 @@
           }
         }
       }
+    },
+    "x-ms-paths": {
+      "/libraries/{libraryName}?comp=appendblock": {
+        "put": {
+          "tags": [
+            "Library"
+          ],
+          "operationId": "Library_Append",
+          "x-ms-examples": {
+            "Append Library": {
+              "$ref": "./examples/Library_Append.json"
+            }
+          },
+          "description": "Append the data to the library resource created using the create operation.",
```

Comment:
Can this be changed to "Append the content to the library resource created using the create operation. The maximum content size is 4MiB.  Content larger than 4MiB must be appended in 4MiB chunks."?

Reply:
okay